.github/workflows/commitlint.yml

```diff
@@ -25,6 +25,7 @@ jobs:
         run: npm install @commitlint/config-conventional @commitlint/cli
 
       - name: Run commitlint on PR title
+        env:
+          PR_TITLE: ${{ github.event.pull_request.title }}
         run: >-
-          echo '${{ github.event.pull_request.title }}' |
-          npx commitlint --verbose
+          echo "$PR_TITLE" npx commitlint --verbose
```

Comment:
It looks like the pipe, `|`, was dropped.

Reply:
good catch